pom.xml

```diff
@@ -2305,26 +2304,6 @@
                 <artifactId>bcprov-jdk15on</artifactId>
                 <version>1.60</version>
             </dependency>
-            <dependency>
```

Comment:
by doing this, you don't drop jackson 1 (that is required by a several other modules) but you let the accidental resolution of deps happen. No doubt it's a bad idea